docs/CONTRIBUTING.md

```diff
@@ -158,3 +160,31 @@ PR is an example of it.
 
 The [adding new BCC-based gadgets in Inspektor Gadget](https://kinvolk.io/blog/2020/04/adding-new-bcc-based-gadgets-in-inspektor-gadget/)
 blogpost presents some more details about this process.
+
+### Updating BCC from upstream
+As is possible to see in `gadget.Dockerfile`, a BCC container image is
+used as parent image for building the gadget container image.
```

Comment:
I'd phrase this in active voice rather than passive voice, and also a little bit more summarized:
As you can see in the `gadget.Dockerfile`, the gadget container image uses the BCC container image as its parent image.

Reply:
I agree, also because it is the only sentence written in passive voice so it is strange to mix both.